editor/gui/editor_run_bar.cpp

```diff
@@ -52,8 +52,20 @@ void EditorRunBar::_notification(int p_what) {
 
 		case NOTIFICATION_THEME_CHANGED: {
 			_update_play_buttons();
+			play_button->add_theme_style_override("disabled", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			play_button->add_theme_style_override("pressed", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
 			pause_button->set_icon(get_editor_theme_icon(SNAME("Pause")));
+			pause_button->add_theme_style_override("disabled", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			pause_button->add_theme_style_override("pressed", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
 			stop_button->set_icon(get_editor_theme_icon(SNAME("Stop")));
+			stop_button->add_theme_style_override("disabled", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			stop_button->add_theme_style_override("pressed", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			play_scene_button->add_theme_style_override("disabled", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			play_scene_button->add_theme_style_override("pressed", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			play_custom_scene_button->add_theme_style_override("disabled", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			play_custom_scene_button->add_theme_style_override("pressed", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			write_movie_button->add_theme_style_override("disabled", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
+			write_movie_button->add_theme_style_override("pressed", get_theme_stylebox(SNAME("normal"), "FlatMenuButton"));
```

Comment:
That's too many overrides...
It would be better to use some theme type variation (a new one if necessary).

Reply:
Will do, I was trying my best to work with what already existed in hopes that it would be less destructive but I can see how it could be simplified with a new variation. I'll add it in my next push, thanks!